app/models/clientsModel.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../database/dbConfig.js');
+
+/**
+ * Retrieves a list of clients who are attending the class with the specified job id 
+ * and are email reminder recipients.
+ * 
+ * @param {*} jobId Id of the job to retrieve clients from
+ * @returns {Promise<void>}
+ */
+async function getByJobId(jobId) {
+    try {
+        return await db('clients')
+            .select('*')
+            .where('job_id', jobId)
+            .andWhere('is_recipient', true);
```

Comment:
In what case would the is_recipient be 0?

Reply:
is_recipient is still a boolean field, it is probably being represented as 0 or 1 in the database.

Reply:
Sorry if my question wasn't clear but I meant to ask when would the is_recipient field ever be switched to false?

Reply:
Oh, my bad. If the client doesn't want to receive class reminder emails then is_recipient would be set to false and therefore there we don't need to retrieve their email id for sending the email.

Reply:
Ohh okok make sense, thanks!